manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -32,6 +32,8 @@ pilot:
 
   env: {}
 
+  affinity: {}
```

Comment:
The operator api is in operator/pkg/apis/istio/v1alpha1/values_types.proto, you also have to modify it.

Reply:
thank you ! 
I modify it !

Reply:
Looks like there's a conflict issue in the proto generated file, could you run `make gen` again and include the changes?